### PR TITLE
Feature/start guess activity with circle

### PR DESCRIPTION
**Pull request by amattei777**

### CODE REVIEW

File: app/src/main/java/com/github/wnder/GuessLocationActivity.java [hunk 1]
```diff
@@ -245,27 +262,40 @@ protected void onDestroy() {
...
         ));
+    }
 
-        //Set camera position
-        CameraPosition position = new CameraPosition.Builder().target(picturePosition).zoom(14).build();
-        mapboxMap.setCameraPosition(position);
+    private double zoomFromKilometers(int kilometers) {
+        int absLat = Math.abs((int) cameraPosition.getLatitude());
+        double latDeformation = 0.00046*Math.pow(absLat, 2);
+        double offset = 13.6 - latDeformation;
```
Comment (rkuenzi-epfl) [inline app/src/main/java/com/github/wnder/GuessLocationActivity.java:295]: Magic numbers

File: app/src/androidTest/java/com/github/wnder/GuessPreviewActivityTest.java [hunk 2]
```diff
@@ -48,10 +52,10 @@ public static void setup(){
     @Test
     public void testGuessLocationButton(){
         Intents.init();
-        onView(withId(R.id.guessButton)).perform(click());
 
         // TODO: Check openGuessActivity() correct execution, probably that the activity to make a guess is actually launched and maybe that it sends the image identifier with it
-        // Intents.intended(hasComponent(GuessActivity.class.getName()));
+        //onView(withId(R.id.guessButton)).perform(click());
+        //Intents.intended(hasComponent(GuessLocationActivity.class.getName()));
```
Comment (rkuenzi-epfl) [inline app/src/androidTest/java/com/github/wnder/GuessPreviewActivityTest.java:58]: Is it still useful?

**Review by leonardvaney — COMMENTED** [review]
Unused imports, use Ctrl + Alt + O to remove them on each class you work on.

**Review by rkuenzi-epfl — APPROVED** [review]
Is ok, but I think there is some useless comments and magic number.

File: app/src/main/java/com/github/wnder/GuessLocationActivity.java (hunk 1 above)
Comment (amattei777) [reply app/src/main/java/com/github/wnder/GuessLocationActivity.java:295]: I'll add a comment but I didn't managed to find documentation for the latitude deformation on zoom.
If you find a better function let me know: https://docs.mapbox.com/help/glossary/zoom-level/

File: app/src/androidTest/java/com/github/wnder/GuessPreviewActivityTest.java (hunk 2 above)
Comment (amattei777) [reply app/src/androidTest/java/com/github/wnder/GuessPreviewActivityTest.java:54]: I should have implemented that test so yes it is still to do.
I didn't by lack of time

**Review by leonardvaney — APPROVED** [review]

**Pull request closed without merging**